### DISCUSSION TITLE
Run app view and notebook editor view together

### DISCUSSION THREAD

**Discussion by aidanmontare-fed** [marimo-team/marimo#2874] [Ideas]
Would it be possible to run the app view alongside the notebook view? That way I can see how my app will look to users while I also work on it's code.

I think either a split screen within marimo's browser tab, or opening this as a second browser tab would both be fine, as long as they could sync together.

I suppose one solution for now is to run a separate a separate instance of `marimo run --watch NOTEBOOK`, which looks like it will work okay. The state isn't shared, but I can at least get a glance of what my app looks like as I'm working.

**Comment by mscolnick** [comment]
`marimo run --watch NOTEBOOK` is the current recommended way

**Comment by liquidcarbon** [comment]
@aidanmontare-fed the `--watch` works but let's build appetite and demand for a little more ;) - see https://github.com/marimo-team/marimo/issues/2910
